### ISSUE TITLE
Rotation period for KMS keys not correctly interpreted (GCP)

### ISSUE THREAD

**Issue by d47zm3** [accurics/terrascan#699]
* terrascan version:
`version: v1.5.0`
* Operating System:
`MacOS BigSur`

### Description

Describe what you were trying to get done.
Tell us what happened, what went wrong, and what you expected to happen.

Valdiating Terrform plan for KMS key with
`"compute","purpose":"ENCRYPT_DECRYPT","rotation_period":"2592000s","self_link"`
results in 
```Violation Details -

        Description    :        Ensure Encryption keys are rotated within a period of 365 days.
        File           :
        Line           :        0
        Severity       :        HIGH
        -----------------------------------------------------------------------

        Description    :        Ensure Encryption keys are rotated within a period of 90 days.
        File           :
        Line           :        0
        Severity       :        MEDIUM
        -----------------------------------------------------------------------
```

Rego policy cannot handle "INTs" it seems
2592000s is lower than 7776000s and violation is still reported

### What I Did

```
Paste the command(s) you ran and the output.
If there was a crash, please include the traceback here.
```

`terrascan scan --iac-type tfplan --iac-file 1.json`

**Comment by fredprod** [comment]
I fixed that using this in accurics.gcp.EKM.139 for example
```
package accurics

checkRotation90Days[api.id]
{
    api := input.google_kms_crypto_key[_]
    to_number(trim(api.config.rotation_period, "s")) > 7776000
}
```
I don't know if it is right method but with this all rotation period under 7776000s (90d) are now valid.

**Comment by amirbenv** [comment]
@harkirat22 @gaurav-gogia is this something we need to fix in accurics.gcp.EKM.139

**Comment by shreyas-phansalkar-189** [comment]
> I fixed that using this in accurics.gcp.EKM.139 for example
> 
> ```
> package accurics
> 
> checkRotation90Days[api.id]
> {
>     api := input.google_kms_crypto_key[_]
>     to_number(trim(api.config.rotation_period, "s")) > 7776000
> }
> ```
> 
> I don't know if it is right method but with this all rotation period under 7776000s (90d) are now valid.

Yeah rotation period needs to be converted to integer before performing a check.
